website/docs/cloud-docs/api-docs/data-retention-policies.mdx

```diff
@@ -0,0 +1,301 @@
+| [200][] | [JSON API document][] (type: ["data-retention-policy-delete-olders"](#delete-older-than-policy) or ["data-retention-policy-dont-deletes"](#dont-delete-policy)) | The request was successful. [Types of Data Retention Policies](#types-of-data-retention-policies). |
+| [404][] | [JSON API error object][]                       | Target resource not found, data retention policy does not exist, or user unauthorized to perform action. |
+
+### Request Body
+
+No request body.
+
+### Sample Request
+
+```shell
+$ curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  https://app.terraform.io/api/v2/organizations/hashicorp/relationships/data-retention-policy
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "drp-Kh74zpKVGNWbpugQ",
+    "type": "data-retention-policy-delete-olders",
+    "attributes": {
+      "delete-older-than-n-days": 60
+    },
+    "relationships": {
+      "target": {
+        "data": {
+          "id": "hashicorp",
+          "type": "organizations"
+        }
+      }
+    }
+  }
+}
+```
+
+## Create or Update Data Retention Policy
+
+This endpoint creates a data retention policy attached to a target resource, or updates the existing policy. 
+
+| Resource      | API Endpoint                                    | 
+| --------------| ----------------------------------------------- |
+| [Workspaces](/terraform/cloud-docs/api-docs/workspaces#create-or-update-data-retention-policy)       | `POST /workspaces/:workspace_id/relationships/data-retention-policy`         |
+| [Organizations](/terraform/cloud-docs/api-docs/organizations#create-or-update-data-retention-policy) | `POST /organizations/:organization_name/relationships/data-retention-policy` |
+| [Site Wide](/terraform/enterprise/api-docs/admin/settings#update-data-retention-policy-settings)     | `POST /admin/data-retention-policy-settings`                                 |
+
+The endpoint can also be used to change the type of the data retention policy on the target. Making a request with the new type of policy will automatically create a data retention policy of the new type and replace the existing policy on the target resource.
+For more information on the types of data retention policies, refer to [Data Retention Policy Types](#types-of-data-retention-policies).
+
+
+| Status  | Response                  | Reason(s)                                                   |
+| ------- | ------------------------- | ----------------------------------------------------------- |
+| [204][] | No Content                | Successfully updated the target resource's data retention policy.   |
+| [404][] | [JSON API error object][] | Target resource not found, or user unauthorized to perform action. |
+
+### Request Body
+
+This `POST` endpoint requires a JSON object with the following properties as a request payload.
```

Comment:
Non-blocking suggestion: Should we replace this period with a colon since we're about to list out the "following properties"?
```suggestion
This `POST` endpoint requires a JSON object with the following properties as a request payload:
```